Banking/src/main/java/com/example/Banking/service/Service.java

```diff
@@ -0,0 +1,33 @@
+package com.example.Banking.service;
+
+import com.example.Banking.dao.UserDao;
+import com.example.Banking.exception.IdNotFoundException;
+import com.example.Banking.model.Account;
+import com.example.Banking.model.UserTransaction;
+import org.springframework.beans.factory.annotation.Autowired;
+
+
+@org.springframework.stereotype.Service
+public class Service {
+
+    @Autowired
+    private UserDao userDao;
```

Comment:
The `TransactionService` should autowire `TransactionDao` and `AccountService` to create the record in transaction table and update the account balance. There is no need for `UserDao`